appinfo/info.xml

```diff
@@ -4,7 +4,7 @@
 	<name>Miro integration</name>
 	<summary>Integration of Miro</summary>
 	<description><![CDATA[Miro integration lets users create boards, share board links and open boards directly in Nextcloud.]]></description>
-	<version>1.0.5</version>
+	<version>2.0.0</version>
```

Comment:
```suggestion
	<version>1.0.6</version>
```
Not a breaking change and no major new marketable feature, so just a patch bump is enough.